### ISSUE TITLE
BLAST crash

### ISSUE THREAD

**Issue by rrwick** [katholt/Kleborate#6]
The `blastx` call in Kleborate can crash and display an error which looks like this:
```
Error: NCBI C++ Exception:
    T0 "/usr/local/easybuild/build/BLAST/2.2.30/vlsci_intel-2015.08.25/ncbi-blast-2.2.30+-src/c++/src/corelib/ncbiobj.cpp", line 926: Critical: ncbi::CObject::ThrowNullPointerException() - Attempt to access NULL pointer.

Traceback (most recent call last):
  File "/vlsci/SG0006/rwick/Kleborate/kleborate/resBLAST.py", line 134, in <module>
    root = ElementTree.fromstring(blast_output)
  File "/usr/local/easybuild/software/Python/2.7.10-vlsci_intel-2015.08.25/lib/python2.7/xml/etree/ElementTree.py", line 1301, in XML
    return parser.close()
  File "/usr/local/easybuild/software/Python/2.7.10-vlsci_intel-2015.08.25/lib/python2.7/xml/etree/ElementTree.py", line 1654, in close
    self._raiseerror(v)
  File "/usr/local/easybuild/software/Python/2.7.10-vlsci_intel-2015.08.25/lib/python2.7/xml/etree/ElementTree.py", line 1506, in _raiseerror
    raise err
xml.etree.ElementTree.ParseError: no element found: line 2119, column 12
```

I ran into this when using BLAST+ v2.2.30 and an assembly with a contig that was nothing but `CCCCCCCCCCCCCCCCCCC...`

**Comment by rrwick** [comment]
BLAST+ v2.6.0 works fine and prevents the issue! So if you encounter this problem, grab a recent BLAST+ release.

I'm not sure exactly which version of BLAST+ fixed this issue, something between 2.2.30 and 2.6.0...

**Comment by Danissss** [comment]
For people who jumps into this issue: blast+ v2.9.0 has the same "Attempt to access NULL pointer" issue; but the v2.6.0 works fine.
